doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,219 @@ Current limitations
+
+To interact with Ceph Daemons, let's deploy toolbox::
+
+        kubectl create -f ./toolbox.yaml
+
+Exec into the rook-ceph-tools pod::
+
+        kubectl -n rook-ceph exec -it $(kubectl -n rook-ceph get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') bash
+
+Check Ceph cluster status::
+
+        [root@minikube /]# ceph -s
+           cluster:
+                id:     3a30f44c-a9ce-4c26-9f25-cc6fd23128d0
+                health: HEALTH_OK
+
+           services:
+                mon: 1 daemons, quorum a (age 14m)
+                mgr: a(active, since 13m)
+                osd: 1 osds: 1 up (since 13m), 1 in (since 13m)
+
+           data:
+                pools:   0 pools, 0 pgs
+                objects: 0 objects, 0 B
+                usage:   5.0 GiB used, 11 GiB / 16 GiB avail
+                pgs:
+
+Scale out monitors::
+
+        [root@minikube /]# ceph orchestrator mon update 3
+        Updated mon count to 3
+
+It takes few minutes for the monitor daemon to come up. Check the status with::
+
+        [root@minikube /]# ceph orchestrator service ls
+        mgr a minikube rook-ceph-mgr-a-55964c5888-w9sbt None None
+        mon a minikube rook-ceph-mon-a-579c967b98-dv7tw None None
+        mon b minikube rook-ceph-mon-b-784b5b5db5-dpnpf None None
+        mon c minikube rook-ceph-mon-c-585d8d5f6b-25v2l None None
+        osd 0 minikube rook-ceph-osd-0-b9666466-r7678 None None
+
+Create a Ceph File System
+-------------------------
+Using ceph-mgr volumes module, we will create a ceph file system::
+
+        [root@minikube /]# ceph fs volume create myfs
+
+By default replicated size for OSD is 3. Since we are using only one OSD. It can cause error. Let's fix this up by setting replicated size to 1.::
+
+        [root@minikube /]# ceph osd pool set cephfs.myfs.meta size 1
+        [root@minikube /]# ceph osd pool set cephfs.myfs.data size 1
+
+Check Cluster status again::
+
+        [root@minikube /]# ceph -s
+          cluster:
+            id:     3a30f44c-a9ce-4c26-9f25-cc6fd23128d0
+            health: HEALTH_OK
+
+          services:
+            mon: 3 daemons, quorum a,b,c (age 27m)
+            mgr: a(active, since 27m)
+            mds: myfs:1 {0=myfs-a=up:active} 1 up:standby-replay
+            osd: 1 osds: 1 up (since 56m), 1 in (since 56m)
+
+          data:
+            pools:   2 pools, 24 pgs
+            objects: 22 objects, 2.2 KiB
+            usage:   5.1 GiB used, 11 GiB / 16 GiB avail
+            pgs:     24 active+clean
+
+          io:
+            client:   639 B/s rd, 1 op/s rd, 0 op/s wr
+
+Create a NFS-Ganesha Server Cluster
+-----------------------------------
+Add Storage for NFS-Ganesha Servers to prevent recovery conflicts::
+
+        [root@minikube /]# ceph osd pool create nfs-ganesha 64
+        pool 'nfs-ganesha' created
+        [root@minikube /]# ceph osd pool set nfs-ganesha size 1
+        [root@minikube /]# ceph orchestrator nfs add mynfs nfs-ganesha mynfs
```

Comment:
A bit confusing here because the name of the nfs cluster and the namespace is the same (mynfs). How about changing the namespace to something else, nfs-namespace maybe? Also how about adding a sentence to explain the above commands. Something along the lines of `creates an nfs cluster "mynfs" in the "nfs-namespace" backed by the OSD pool "nfs-ganesha" `